bundles/io/org.openhab.io.transport.cul/src/main/java/org/openhab/io/transport/cul/internal/AbstractCULHandler.java

```diff
@@ -252,8 +253,10 @@ protected void processNextLine(String data) {
             return;
         } else if (data.matches("^\\d+\\s+\\d+")) {
             processCreditReport(data);
+            return;
```

Comment:
This return prevents "notifyDataReceived" is called. Doing so, the SendingThread will wait until Timeout (2s).

Reply:
Before #5329 it works that way, I think that credit data is special that don't need to notify